app/src/androidTest/java/com/sdp/movemeet/view/map/MainMapFragmentTest.java

```diff
@@ -84,16 +75,10 @@
 
 
     @Before
-    public void setUp() throws InterruptedException {
-        //The map is tested when the user is logged in
+    public void setUp() {
         fAuth = FirebaseAuth.getInstance();
-        fAuth.signInWithEmailAndPassword("test@test.com", "password").addOnCompleteListener(new OnCompleteListener<AuthResult>() {
-            @Override
-            public void onComplete(@NonNull Task<AuthResult> task) {
-                user = fAuth.getCurrentUser();
-                uiDevice = UiDevice.getInstance(getInstrumentation());
-            }
-        });
+        fAuth.signInWithEmailAndPassword("test@test.com", "password");
```

Comment:
Why we didn't need the onCompleteListener anymore ?

Reply:
The instanciation of the UiDevice is not dependant from the user being logged in, so there is no need to have it in the onCompteListener. And now that the listener is not doing anything, I could just delete it.